machines/bleShare/scan/scanMachine.ts

```diff
@@ -723,6 +732,9 @@ export const scanMachine =
     },
     {
       actions: {
+        offlineaccepted: () => console.log('>> offline QR accepted'),
```

Comment:
do we need to log the QR acceptance or rejection , which would be seen in the UI itself, and moreover this log is more generic.